### ISSUE TITLE
[Bug][a11y] Site Permission-Touch target-Site permissions statuses need to have the touch target resized

### ISSUE THREAD

**Issue by Diana-Rus** [mozilla-mobile/fenix#9778]
## Prerequisites
Have Google Accessibility Scanner installed.
### Steps to reproduce
1. Open app and navigate to "permission.site".
2. Enable the "https" version.
3. Tap on one of the options that generates the site permission pop up such as Location.
4. Check the "Remember decision for this site" and tap on "allow".
5. Tap the "Lock" icon for the site.
6. Scan screen.
### Expected behavior
There are no suggestions to make any changes.
### Actual behavior
Site permission details - Location status
- org.mozilla.fenix.nightly:id/locationStatus
This item's height is 28dp. Consider making the height of this touch target 48dp or larger.
### Device information

* Android device: Samsung Galaxy S9 (Android 8)
* Fenix version: Nightly 4/7 (Build #20980610)

Note:
- The same actual result is present for "Camera" or "Microphone" status field the height is 28 dp.
- It needs to be taken into consideration that when one site contains multiple permissions, the space between the saved permissions the height is limited please check last screen.

**Screen:**
**Site permission details - "Location" status**
![Screenshot_20200407-184516_Accessibility Scanner](https://user-images.githubusercontent.com/49859863/78690857-a7c6b580-7900-11ea-9fda-e8b0ac5aaba7.jpg)

**Site permission details**
![screenshot_Firefox Nightly_2020-04-07-18_44_46-edit](https://user-images.githubusercontent.com/49859863/78691287-1e63b300-7901-11ea-8cea-5b0f2063b7d2.png)

**Comment by Mugurell** [comment]
Increasing touch target will mean increasing the size of the items so the layout will change.
Asking for UX feedback if this should be done or not.

**Comment by AmyYLee** [comment]
Assigning to @brampitoyo

**Comment by brampitoyo** [comment]
@Mugurell If we increase it to 48dp as recommended, would you be able to post a screenshot?

It’s likely going to be fine for the layout. We just need to double-check and make sure that it doesn’t break anything or introduce awkward empty spaces.

**Comment by Mugurell** [comment]
> @Mugurell If we increase it to 48dp as recommended, would you be able to post a screenshot?
> 
> It’s likely going to be fine for the layout. We just need to double-check and make sure that it doesn’t break anything or introduce awkward empty spaces.

It would indeed look better:


<table>
  <tr>
     <td>Current</td>
     <td>Proposed</td>
  </tr>
  <tr>
    <td><img src="https://user-images.githubusercontent.com/11428869/88507044-29dc0680-cfe4-11ea-8c57-08e83bf3b56b.png" width=270 height=540></td>
    <td><img src="https://user-images.githubusercontent.com/11428869/88507061-37918c00-cfe4-11ea-9776-bbdc124691c7.png" width=270 height=540></td>
  </tr>
 </table>

**Comment by brampitoyo** [comment]
@Mugurell Looks great to me!
